src/main/java/net/codejava/MvcController.java

```diff
@@ -311,6 +319,18 @@ public String submitWithdraw(@ModelAttribute("user") User user) {
                                                     userWithdrawAmtInPennies);
       jdbcTemplate.update(transactionHistorySql);
 
+      // calculate points 

```

Comment:
Edit javadoc comment for this method as well.
Also, in addition to your "// calculate points", add comments explaining both the rewardsToAdd variable and the leftover variable created on lines 325-326. Just a few words/a sentence should do

Reply:
Done